packages/integrations/image/src/index.ts

```diff
@@ -1,5 +1,116 @@
-import integration from './integration.js';
-export * from './lib/get-image.js';
-export * from './lib/get-picture.js';
+import type { AstroConfig, AstroIntegration } from 'astro';
+import { ssgBuild } from './build/ssg.js';
+import { ssrBuild } from './build/ssr.js';
+import { PKG_NAME, ROUTE_PATTERN } from './constants.js';
+import { ImageService, TransformOptions } from './loaders/index.js';
+import { filenameFormat, propsToFilename } from './utils/paths.js';
+import { createPlugin } from './vite-plugin-astro-image.js';
 
-export default integration;
+export { getImage } from './lib/get-image.js';
```

Comment:
Turns out moving the actual integration out of `index.ts` was sometimes confusing the editor when working with `astro.config.mjs` files

Reply:
Hmm, not sure if I understand properly, but is there anything actionable for me here?